.github/workflows/test-reset.yml

```diff
@@ -11,7 +11,7 @@ permissions:
 
 jobs:
   call-reset:
-    uses: TrafficDriven/ml-deployer/.github/workflows/reset.yml@main
+    uses: TrafficDriven/ml-deployer/.github/workflows/reset.yml@dev
```

Comment:
do you want to push a dev tag to main?

Reply:
yeah, these are the test workflows that were setup; having them point to main tests what we have now and not the changes we made